include/veriblock/blockchain/pop/vbk_block_tree.hpp

```diff
@@ -83,6 +92,7 @@ struct VbkBlockTree : public BlockTree<VbkBlock, VbkChainParams> {
                           bool isBootstrap = false) override;
 
   PopForkComparator cmp_;
+  PayloadsStorage& storagePayloads_;
```

Comment:
we should probably change this to `std::shared_ptr<PayloadsStorage> storagePayloads_`

Reply:
same for ALT